src/sentry/models/deploy.py

```diff
@@ -85,3 +93,10 @@ def notify_if_ready(cls, deploy_id, fetch_complete=False):
             if activity is not None:
                 activity.send_notification()
                 deploy.update(notified=True)
+                # XXX(workflow): delete this after WF 2.0 experiment over
+                org = Organization.objects.filter(id=deploy.organization_id).first()
```

Comment:
Are you able to use `Organization.objects.get_from_cache()` instead? You should generally use `get()` instead of `first()` as first() [will add an order clause](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#first) which can yield poorer performance than `get()` will.

Given that you are mostly using the organization to check features reading from cache will save us a SQL query.